http/errors_test.go

```diff
@@ -0,0 +1,55 @@
+package http
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_IsOTPRequired(t *testing.T) {
```

Comment:
suggestion(non blocking): it really looks like this test would be better with table tests

Reply:
Yep why not